reconcile/aws_saml_roles/integration.py

```diff
@@ -58,7 +58,7 @@ class AwsSamlRolesIntegrationParams(PydanticRunParams):
     log_cached_log_output: bool = False
 
     @validator("max_session_duration_hours")
-    def max_session_duration_range(cls, v: str | int) -> int:
+    def max_session_duration_range(cls, v: str | int) -> int:  # noqa: N805
```

Comment:
please update pep8 config instead of comment on each validator usage

```toml
[tool.ruff.pep8-naming]
classmethod-decorators = ["classmethod", "pydantic.validator"]
```